src/decorators.py

```diff
@@ -121,9 +121,25 @@ def __exit__(self, exc_type, exc_value, tb):
 
         variables[1] = _local.handler.traceback
 
-        if not botconfig.PASTEBIN_ERRORS or channels.Main is not channels.Dev:
+        if var.REPORT_ERRORS:
+            api_url = "https://werewolf.chat/devsite/report.php"
+            api_body = logger.get_replay(source)
```

Comment:
This should be inside a `with _local.handler:` block. It's unlikely that it does fail, but if it does this may potentially crash the bot and wipe out any log. I'm not sure what we should do if it does fail, but probably continue without any form of replay.